person.rb

```diff
@@ -1,16 +1,20 @@
 require_relative 'nameable'
-require_relative 'decorator'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     super()
     @id = Random.rand(1...1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
   end
```

Comment:
- [ ] It would be better if the person itself is sent as a parameter to create the new rental achieving the `has-many` association. For example; `Rental.new(date, book, self)`

Reply:
Thanks, @addod19 Great tip.